content/news/25_05_20_Berd_WF/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Analysing data from BERD Data Portal with Galaxy
+date: '2025-05-20'
+tease: "Find out how to research your Data from the BERD Data Portal directly in Galaxy"
+continent: EU
+contact: Daniela Schneider
+gtn: false
+tags: [news]
+subsites: [all]
+---
+In April, we included the [Berd4NFDI](https://www.berd-nfdi.de/about/) data portal in [Galaxy](https://galaxyproject.org/news/2025-04-09-berd-integration/). 
+BERD is a NFDI consortium working towards a platform for collecting, (pre-)processing, analysing and preserving Business, Economic and Related Data. 
+Their data portal is a hub for sharing suitable datasets.
+
+## Getting started in Galaxy
+
+To analyse BERD data directly in Galaxy, go to the [Galaxy platform](https://usegalaxy.eu/). 
+On the left pane, click on the "Upload" button. In the new window, click on the button 'Choose from repository'. 
+In the opening window, insert "BERD" in the search bar. Now you can click on the folder "BERD@NFDI Repository". 
+The datasets from BERD's portal show up here. You can select any folder to see its content. 
+To choose one, click on the item or items you want to work with and click on the 'Select' button in the corner. Click on 'Start' and close this window.
+We worked with "Instagram Posts from German Parties" and clicked it to upload the CSV file.
+
+## Example Analysis
+
+The [dataset](https://berd-platform.de/records/nghbn-9gx74) "Instagram Posts from German Parties" is particularly insightful as the file contains posts from 632 of the 1134 people standing for election in the Bavarian state election.  
+Since the data is messy, we first open it in the interactive environment of Open Refine, directly from Galaxy. 
+
+![Screenshot of the BERD  example file in Open Refine on Galaxy](content/news/25_05_20_Berd_WF/SC_Galaxy_OpenRefine.png "Example file in Open Refine on Galaxy")
+You can adapt the data in OpenRefine according to your needs. In the next step, we export the file out of the interactive environment, back to the Galaxy platform.
+There, you can add further analysis, like removing line breaks with Regular Expressions. 
+In this example, we were interested in two things: the most prevalent themes of the posts and where they originated.
+Therefore, we first cut out only the posts from the table, cleaned the content from stopwords and made a word cloud.
```

Comment:
maybe move this sentence up at the very beginning? In this PoC we will demonstrate how you can use data from BERD, do some data cleaning and wrangling and plot a wordcloud and a geo map ....